auth/src/browser-store.ts

```diff
@@ -0,0 +1,89 @@
+import * as ucan from 'ucans'
+import AuthStore from './auth-store.js'
+import * as builders from './builders.js'
+
+// @TODO: Stand in since we don't have actual root keys
+// ---------------
+const PRIV_KEY =
+  'aghhYelL58I8vwVWt7W5cs+Y4uYYWQma6jX0MjkCt7cOdiciXp5JJcOtoWHCgljZhLFV/h0E9zdeYXoSZ0rCgA=='
+
+const isRootKey = (): boolean => {
+  console.log(process.env.REACT_APP_ROOT_KEY)
+  return process.env.REACT_APP_ROOT_KEY === 'true'
```

Comment:
Is there any way this can be an API that's exposed by the auth lib, rather than the auth lib directly referenceing process.env?

Reply:
ah yeah you're right. that's an artifact of refactoring this out of the app layer. I'll handle env vars in the react app 👍